### ISSUE TITLE
don't report variable as unused if it's global

### ISSUE THREAD

**Issue by MarcoGorelli** [MarcoGorelli/cython-lint#44]
e.g.
```
cdef int current_notification = FALLBACK_QUIETLY

def set_fallback_notification(level):
    global current_notification
    current_notification = level
```

**Comment by MarcoGorelli** [comment]
fixed now
